File: app/src/main/java/org/jellyfin/androidtv/ui/playback/AudioEventListener.java
package org.jellyfin.androidtv.ui.playback;

import org.jellyfin.apiclient.model.dto.BaseItemDto;

public class AudioEventListener {
    public void onPlaybackStateChange(PlaybackController.PlaybackState newState, BaseItemDto currentItem) {}
    public void onProgress(long pos) {}
    public void onQueueStatusChanged(boolean hasQueue) {}
    public void onQueueReplaced(){}
}

File: app/src/main/java/org/jellyfin/androidtv/ui/playback/AudioEventListener.kt
package org.jellyfin.androidtv.ui.playback

import org.jellyfin.sdk.model.api.BaseItemDto

interface AudioEventListener {
	fun onPlaybackStateChange(newState: PlaybackController.PlaybackState, currentItem: BaseItemDto?) = Unit
	fun onProgress(pos: Long, duration: Long) = Unit
	fun onQueueStatusChanged(hasQueue: Boolean) = Unit
	fun onQueueReplaced() = Unit
}
